.openshift-ci/gcp/command.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+set -e
+
+runCommand() {
+    local GCP_VM_NAME="$1"
+    shift
+    local GCP_VM_COMMAND="$1"
+    shift
+
+    [ -z "$GCP_VM_NAME" ] && \
+        echo "error: missing parameter GCP_VM_NAME" && return 1
+
+    [ -z "$GCP_VM_COMMAND" ] && \
+        echo "error: missing parameter GCP_VM_COMMAND" && return 1
+
+    success=false
+    for _ in {1..3}; do
+        if gcloud compute ssh "$GCP_VM_NAME" --command "cd kernel-packer; $GCP_VM_COMMAND"; then
+            success=true
+            break
+        else
+            echo "Retrying in 5s ..."
+            sleep 5
+        fi
+    done
+
+    if test ! "$success" = "true"; then
+        echo "Failed to run command after 3 retries"
+        return 1
+    fi
+
+    return 0
+}
+
+main() {
+    local GCP_VM_NAME="$1"
+    shift
+    local GCP_VM_COMMAND="$1"
+    shift
+
+    which gcloud || true
+
+    echo "Running command $GCP_VM_COMMAND..."
+    runCommand "$GCP_VM_NAME" "$GCP_VM_COMMAND"
+}
+
+COMMAND=$1
```

Comment:
Maybe we can use `$@` instead of `$1`, that way we could have commands with more than one parameter if we ever need them.